plugins/inputs/fluentd/README.md

```diff
@@ -0,0 +1,53 @@
+# Fluentd Input Plugin
+
+The fluentd plugin gathers metrics from plugin endpoint provided by [in_monitor plugin](http://docs.fluentd.org/v0.12/articles/monitoring).
+This plugin understands data provided by /api/plugin.json resource (/api/config.json is not covered).
+
+### Configuration:
+
+```toml
+# Read metrics exposed by fluentd in_monitor plugin
+[[inputs.fluentd]]
+	## This plugin only reads information exposed by fluentd using /api/plugins.json.
+	## Tested using 'fluentd' version '0.14.9'
+	##
+	## Endpoint: 
+	## - only one URI is allowed
+	## - https is not supported
+	endpoint = "http://localhost:24220/api/plugins.json"
+	
+	## Define which plugins have to be excluded (based on "type" field - e.g. monitor_agent)
+	exclude = [
+		"monitor_agent",
+		"dummy",
+	]
+```
+
+### Measurements & Fields:
+
+Fields may vary depends on type of the plugin
+
+- fluentd
+    - RetryCount            (float, unit)
+    - BufferQueueLength     (float, unit)
+    - BufferTotalQueuedSize (float, unit)
+
+### Tags:
+
+- All measurements have the following tags:
+	- PluginID        (unique plugin id)
+	- PluginType      (type of the plugin e.g. s3)
+    - PluginCategory  (plugin category e.g. output)
```

Comment:
Telegraf normally uses snake_case for tag and field names, can you update these to use it?